docs/overview.md

```diff
@@ -46,7 +44,7 @@ The following procedure provides an overview with links to details for every ste
   is performed automatically, it is a good idea to ensure
   that changes on one node actually replicate to other nodes.
 
-* [Install ProxySQL](howtos/proxysql.md#load-balancing-with-proxysql).
+* [Install ProxySQL](proxysql.md#load-balancing-with-proxysql).
 
   To complete the deployment of the cluster, a high-availability proxy is required. We recommend installing [ProxySQL](https://www.proxysql.com/) on client nodes for efficient workload management across the cluster without any changes to the applications that generate queries.
 
```

Comment:
We should change the title of this doc. Since now we have two docs with the title "Quick start guide"